src/components/Submit/__tests__/index.spec.tsx

```diff
@@ -4,6 +4,9 @@ import Submit from '..'
 import { shouldMatchEmotionSnapshot } from '../../../helpers/jestHelpers'
 import mockErrors from '../../../mocks/mockErrors'
 import Form from '../../Form'
+import TextBoxField from '../../TextBoxField'
+
+const alpha = /^[a-zA-Z]*$/
```

Comment:
Don't we have this regex in `@scaleway/regex`?

Reply:
Yes but we don't have it imported and that would be a bit overkill to add it just for some tests :)